eks-anywhere-common/Testers/Kong/enterprise-testJob.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kong-enterprise-testjob
+  namespace: kong
+spec:
+  backoffLimit: 1
+  template:
+    spec:
+      containers:
+        - name: job
+          image: curlimages/curl
+          command: ["curl", "-ks", "--location", "http://eks-anywhere-kong-proxy.kong.svc.cluster.local:80"]
```

Comment:
Please check our [Functional Job Requirements](https://aws-samples.github.io/eks-anywhere-addons/#functional-job-requirements). `curl` is not an approved test job for functional test.

Reply:
hi @elamaran11 we provided a more robust functional test. 

Regarding the question on keys, yes this test does work without a license key, it will run in evaluation mode. 
https://docs.konghq.com/gateway/latest/licenses/